src/ui/components/filters/sortoptionscomponent.js

```diff
@@ -46,6 +46,9 @@ export default class SortOptionsComponent extends Component {
   }
 
   onMount () {
+    if (this.getState('resultsContext') === ResultsContext.NO_RESULTS) {
+      return;
+    }
```

Comment:
Ooh what do you think about adding null checks for the selectors before setting the on-click listeners?

Reply:
oo at first I was like "but I kind of like how there's no safety net and I will get console errors if I make a mistake". Then I realized I could just add a unit test